File: php/lib/Timer.php
<?php

namespace lib;

class Timer {
	
  private $start = 0;	
	
  public function __construct() {
    $this->start = \microtime(true);		
  }
	
  public function reinit() {
    $this->start = \microtime(true);
  }
	
  public function startSince() {
    $time = round(\microtime(true) - $this->start, 2);
		
    return $time;
  }	
}

File: java/src/perslib/Timer.java
package perslib;

public class Timer {
	
	private long m_start = 1L;	
	static int nbtimer = 1;
	
	public static Timer init() {
		
		return new Timer();
	}
	
	public void reinit() {
		m_start = System.currentTimeMillis();
	}
	
	public Timer() {
		m_start = System.currentTimeMillis();		
		nbtimer++;
	}
	
	public double startSince() {
		double temps = perslib._Math.round((((double) System.currentTimeMillis() - m_start) / 1000.0), 2);
		
		return temps;
	}	
}
